src/libraries/icubmod/canBusVirtualAnalogSensor/canBusVirtualAnalogSensor.ini

```diff
@@ -1,5 +0,0 @@
-[plugin canBusVirtualAnalogSensor]
-type device
-name canBusVirtualAnalogSensor
-library canBusVirtualAnalogSensor
-wrapper virtualAnalogServer
```

Comment:
Warning: the `yarp_prepare_plugin` method uses `grabber`

Reply:
fixed to virtualAnalogServer